### PR TITLE
STITCH-1736: add run_as_system to api for executing debug function

### PR DESCRIPTION
**Pull request by christineschen**
some things may be broken until the backend changes get merged

### CODE REVIEW

**Comment by edaniels** [comment]
core owns 3.x branch for this so need to have SDK team review 😎

File: src/admin.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-import 'fetch-everywhere';
-import { newStitchClient, StitchClient } from './client';
-import ADMIN_CLIENT_TYPE from './common';
-import { ADMIN_CLIENT_CODEC } from './auth/common';
-import { StitchError } from './errors';
+import "fetch-everywhere";
+import { newStitchClient, StitchClient } from "./client";
+import ADMIN_CLIENT_TYPE from "./common";
+import { ADMIN_CLIENT_CODEC } from "./auth/common";
+import { StitchError } from "./errors";
```
Comment (mburdette) [inline src/admin.js]: I think your prettier added double quotes, should just leave these as single.

File: src/admin.js [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
 
 const v3 = 3;
 
 /** @private **/
 export class StitchAdminClientFactory {
   constructor() {
-    throw new StitchError('StitchAdminClient can only be made from the StitchAdminClientFactory.create function');
+    throw new StitchError(
+      "StitchAdminClient can only be made from the StitchAdminClientFactory.create function"
```
Comment (mburdette) [inline src/admin.js]: same here

File: src/admin.js [hunk 3]
```diff
@@ -1,21 +1,26 @@
...
+    );
   }
 
   static create(baseUrl) {
-    return newStitchClient(StitchAdminClient.prototype, '', {baseUrl, authCodec: ADMIN_CLIENT_CODEC});
+    return newStitchClient(StitchAdminClient.prototype, "", {
```
Comment (mburdette) [inline src/admin.js]: and here

**Pull request closed without merging**